test/fixtures/sample-project/src/graphql/query-with-duplicate-fragments.graphql

```diff
@@ -0,0 +1,6 @@
+query FancyQuery($id: ID!) {
+  node(id: $id) {
+    ...ProductFragment
+    ...ProductFragment
+  }
```

Comment:
This is valid graphql, and it does generate duplicate fragment declarations before the deduping logic was added, so I know the test is valid 😄